### PR TITLE
feat(config-ui): move Dashboard from sidebar to the top right corner

### LINKED ISSUE #4655: [Feature][Config UI] Move Dashboard from sidebar to the top right corner
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-devlake/issues?q=is%3Aissue) and found no similar feature requirement.

### Use case

_No response_

### Description

Now:
Screen Shot 2023-03-14 at 2 43 02 PM

New:
Screen Shot 2023-03-14 at 2 42 52 PM
Prototype: https://www.figma.com/file/Eux4Dhorxp7M0TwzOnL6Yh/New-Config-UI-(wym)?node-id=1996%3A11140&t=KkgT61M6mNGz3MoM-1

### Related issues

_No response_

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by coldgust**
### ⚠️ Pre Checklist

> Please complete _ALL_ items in this checklist, and remove before submitting

- [x] I have read through the [Contributing Documentation](https://devlake.apache.org/community/).
- [ ] I have added relevant tests.
- [ ] I have added relevant documentation.
- [ ] I will add labels to the PR, such as `pr-type/bug-fix`, `pr-type/feature-development`, etc.

### Summary
move Dashboard from sidebar to the top right corner

### Does this close any open issues?
Closes #4655 

### Screenshots
Snipaste_2023-03-15_19-32-11

### CODE REVIEW

File: config-ui/src/layouts/base/base.tsx [hunk 1]
```diff
@@ -93,6 +101,11 @@ export const BaseLayout = ({ children }: Props) => {
       <S.Inner>
         <S.Header>
           <Navbar.Group align={Alignment.RIGHT}>
+            <a href={getGrafanaUrl()} rel="noreferrer" target="_blank">
```
Comment (mintsweet) [inline config-ui/src/layouts/base/base.tsx:104]: Can you add a border to it, like a design draft?

**Review by mintsweet — CHANGES_REQUESTED** [review]
Thank you so much for your `devlake` contributions.

File: config-ui/src/layouts/base/base.tsx (hunk 1 above)
Comment (coldgust) [reply config-ui/src/layouts/base/base.tsx]: I have added a border.
Snipaste_2023-03-16_22-20-55

Comment (mintsweet) [reply config-ui/src/layouts/base/base.tsx]: cool.

**Review by mintsweet — APPROVED** [review]
LGTM.

**Review by likyh — APPROVED** [review]

**Pull request merged**